src/ntfs/read.rs

```diff
@@ -110,7 +110,13 @@ pub fn manage_fares_v1(collections: &mut Collections, base_path: &path::Path) ->
     let file_od_fares = "od_fares.csv";
     let file_fares = "fares.csv";
 
-    if !base_path.join(file_prices).exists() || !base_path.join(file_od_fares).exists() {
+    if !base_path.join(file_prices).exists()
+        || !base_path.join(file_od_fares).exists()
+        || (!collections.tickets.is_empty()
```

Comment:
Just to make the code more self-documenting about what you're doing, I would extract a function `has_fares_v2()`.
```rust
fn has_fares_v2(collections: &Collections) -> bool {
	!(collections.tickets.is_empty() &&
		collections.ticket_prices.is_empty() &&
		collections.ticket_uses.is_empty() &&
		collections.ticket_use_perimeters.is_empty() &&
		collections.ticket_use_restrictions.is_empty())
}
```

And you can then use it
```rust
if !base_path.join(file_prices).exists()
        || !base_path.join(file_od_fares).exists()
        || !has_fares_v2(collections)
```